### PR TITLE
Implement seminar functionalities

### PR DESCRIPTION
**Pull request by harithadiv**
- Event type seminar introduced 
- parsers and commands
- junit

### CODE REVIEW

**Review by lowjiayee — COMMENTED** [review]
Is it possible to remove logs from your commit?

File: build.gradle [hunk 1]
```diff
@@ -14,6 +14,7 @@ repositories {
 
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter:5.5.0'
+    implementation 'org.jetbrains:annotations:15.0'
```
Comment (benchan911) [inline build.gradle]: Why the gradle file is change? Is there a reason for it?

Comment (benchan911) [inline build.gradle:17]: Is there a reason why the gradle file is updated?

File: src/main/java/seedu/command/event/ListEvent.java [hunk 2]
```diff
@@ -14,11 +14,6 @@ public ListEvent(EventList eventList) {
 
     @Override
     public void execute() throws DukeException {
-        if (eventList.getSize() == 0) {
-            throw new DukeException("List is empty");
-        }
-        for (int i = 0; i < eventList.getSize(); i++) {
-            System.out.println(i + 1 + ". " +  eventList.find(i));
-        }
+        eventList.listEvent();
```
Comment (benchan911) [inline src/main/java/seedu/command/event/ListEvent.java]: Looks good

**Review by benchan911 — COMMENTED** [review]
I think it looks good, can merge

**Review by lowjiayee — APPROVED** [review]

**Pull request merged**